packages/base/src/components/Section.tsx

```diff
@@ -29,16 +29,22 @@ export const Section: React.SFC<SectionProps> = (props) => (
         <div className="g-section">
             <div className={`panel panel-default ${props.panelClassName ? props.panelClassName : ''}`}>
                 <div className="panel-body">
-                    <div style={{borderBottom: '2px solid #cccccc', marginBottom: '30px'}}>
-                        <div style={{display: 'inline-block', fontSize: '200%', marginBottom: '8px'}}>{props.title}</div>
-                        {props.context && (
+                    <div style={props.title ? {borderBottom: '2px solid #cccccc', marginBottom: '30px'} : {}}>
+                        {props.title &&
+                            <div style={{display: 'inline-block', fontSize: '200%', marginBottom: '8px'}}>
```

Comment:
Any reason not to use CSS classes here instead of inline style?

Reply:
Nope, no reason. This Section component came over from biologics to glass this way, and still has the comment above the component saying (I just didn't want to touch that part in this fb):
// FIXME: remove all of these inline styles, make actual CSS classes.